README.md

```diff
@@ -529,11 +529,14 @@ This repository contains a curated list of awesome open source libraries that wi
 
 
 ## Commercial Platform
+* [Amazon Augmented AI](https://aws.amazon.com/augmented-ai/) - Amazon Augmented AI (A2I) allows you to conduct a human review of machine learning (ML) systems to guarantee precision.
```

Comment:
I'd be keen to merge all AWS to one, ideally single product per company in this section, we may need to update the contribting file

Reply:
sorry that is typo

Reply:
I wonder how we can unite all AI-related products into one, may use Amazon directly? @axsaucedo